src/workloads/scale/LargeIndexedIns.yml

```diff
@@ -0,0 +1,78 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload sends bursts of finds that use large $ins while running
+  a low rate update workload.  The workload causes high CPU load as the server
+  becomes bottlenecked on tcmalloc spinlocks during the find operations.
+  Improvements or regressions in this aspect of the allocator should
+  be measurable by the average CPU usage during this test as well as
+  the latency of the find operations.
+
+  In this workload, large arrays of random strings are generated to use
+  for the $in queries.  To avoid a CPU bottleneck on the workload client,
+  it uses a ^Once generator to generate the arrays once during initialization.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 20000
+
+Actors:
+- Name: Loader
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: largeins
+    CollectionCount: 1
+    Threads: 1
+    DocumentCount: 1000
+    BatchSize: 1000
+    Indexes:
+    - keys: {"hash" : 1, "key" : 1, "string1" : 1, "string2" : 1}
+    - keys: {"key": 1}
+    Document:
+      _id: {^Inc: {start: 0}}
+      hash: &randomString {^RandomString: {length: 22}}
+      key: *randomString
+  - {Nop: true}
+
+- Name: FindLargeIn
+  Type: CrudActor
+  Threads: 64
+  Database: largeins
+  Phases:
+  - {Nop: true}
+  - Name: FindLargeIn
+    Duration: 2 minutes
+    GlobalRate: 1000 per 15 seconds
+    Collection: Collection0
+    Operations:
+    - OperationName: find
+      OperationCommand:
+        Filter:
+          hash:
+            $in: &arrayGenerator {^Once: {^Array: {of: *randomString, number: 20000}}}
+          key:
+            $in: *arrayGenerator
+
+- Name: Update
+  Type: CrudActor
+  Database: largeins
+  ClientName: Update
+  Threads: 10
+  Phases:
+  - {Nop: true}
+  - Duration: 2 minutes
+    RecordFailure: true
+    GlobalRate: 1 per 250 milliseconds
+    Collection: Collection0
+    Operations:
+    - OperationName: updateOne
+      OperationCommand:
+        Filter: {id: {^RandomInt: {min: 0, max: 1000}}}
+        Update:
+          $inc: {a: 1}
+
+AutoRun:
+  Requires:
+    mongodb_setup: [replica]
```

Comment:
I *think* this is outdated now with the recent changes to `AutoRun` (ping vlad) so this may not run / may barf when merged.

Reply:
Hey @vrachev, do you know if this is the correct way to use AutoRun?

Reply:
Yes this is the old way. Take a look at the [new syntax](https://github.com/mongodb/genny/tree/bfaff11cabfa9e9f362458f597943259e754f1f1#patch-testing-genny-changes-with-sys-perf--dsi), additionally you can see what the other yamls are doing.

Also note that with the new syntax this workload will run on every mongodb version (4.0 -> master) unless specifically prohibited. Please add a condition around `branch_name` for all branches that should not/cannot run this workload. This is explained more thoroughly in the readme.

Even more context here: https://mongodb.slack.com/archives/C01VD0LQZED/p1623354659212300

Reply:
Thanks!  Updated to use the new syntax